Classes/Controller/AbstractBackendController.php

```
@@ -2036,4 +2093,51 @@ protected function getTemplateConfigurations(): array
+                continue;
+            }
+
+            $allowedTables = GeneralUtility::trimExplode(',', $GLOBALS['TCA'][$this->getTableName()]['columns'][$column['columnName']]['config']['allowed'] ?? '');
+            if (count($allowedTables) > 1) {
+                // @TODO: Resolve multiple table relations
+                continue;
+            }
+
+            $foreignTable = $GLOBALS['TCA'][$this->getTableName()]['columns'][$column['columnName']]['config']['foreign_table'] ?? null;
+            $mmTable = $GLOBALS['TCA'][$this->getTableName()]['columns'][$column['columnName']]['config']['MM'] ?? null;
+            $mmOppositeField = $GLOBALS['TCA'][$this->getTableName()]['columns'][$column['columnName']]['config']['MM_opposite_field'] ?? null;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$mmOppositeField'. (undefined)

(UnusedLocalVariable)